src/applications/disability-benefits/all-claims/components/ComboBox.jsx

```diff
@@ -5,7 +5,7 @@ import { VaTextInput } from '@department-of-veterans-affairs/component-library/d
 import { fullStringSimilaritySearch } from 'platform/forms-system/src/js/utilities/addDisabilitiesStringSearch';
 
 const COMBOBOX_LIST_MAX_HEIGHT = '440px';
-
+const defaultHighlightedIndex = -1;
```

Comment:
Note to reviewer: 
This line is what makes the focus default to not highlighting any of the autosuggestions